chains/ethereum/deposit.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package ethereum
+
+import (
+	"math/big"
+
+	erc20Mintable "github.com/ChainSafe/ChainBridgeV2/bindings/ERC20Mintable"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+func mintErc20Tokens(connection *Connection, opts *bind.TransactOpts, contractAddress, recipient common.Address, amount *big.Int) error {
```

Comment:
Most of these methods should live inside a test file, they're not something the bridge would actually need to call afaik

Reply:
Right on, I guess thats the difference between have a CLI for this stuff or not